backend/Platform/settings/base.py

```diff
@@ -194,10 +194,11 @@
 # Enable admin login through shibboleth
 SHIB_ADMIN = True
 
-# Other settings
-EMAIL_WEB_SERVICE_URL = os.environ.get("EMAIL_WEB_SERVICE_URL", "http://127.0.0.1")
-EMAIL_WEB_SERVICE_USERNAME = os.environ.get("EMAIL_WEB_SERVICE_USERNAME", "")
-EMAIL_WEB_SERVICE_PASSWORD = os.environ.get("EMAIL_WEB_SERVICE_PASSWORD", "")
+# Email web service
+EMAIL_OAUTH_CLIENT_ID = os.environ.get("EMAIL_OAUTH_CLIENT_ID", "")
+EMAIL_OAUTH_CLIENT_SECRET = os.environ.get("EMAIL_OAUTH_CLIENT_SECRET", "")
+EMAIL_OAUTH_TOKEN_URL = os.environ.get("EMAIL_OAUTH_TOKEN_URL", "")
+EMAIL_OAUTH_API_URL_BASE = os.environ.get("EMAIL_OAUTH_API_URL_BASE", "")
```

Comment:
Should we have a fallback for these just like before

Reply:
There's no local equivalent so it'll just fail, which I think is okay.